crates/iceberg-catalog/src/implementations/postgres/tabular/table/mod.rs

```diff
@@ -128,77 +128,6 @@ impl From<DbTableFormatVersion> for FormatVersion {
     }
 }
 
-pub(crate) async fn load_tables_old(
```

Comment:
We should also remove `only_unmigrated: bool,` from `tabular::list_tabulars`

Reply:
Done, I also removed the column from `tabulars`